components/ReviewCard.tsx

```diff
@@ -1,20 +1,35 @@
 import React, { useState } from 'react'
+
```

Comment:
?

Reply:
I was trying to group the imports to appear nicer? lol